native/components/message-markdown.react.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import * as SimpleMarkdown from 'simple-markdown';
+
+import { getMarkdownStyles } from '../themes/markdown-styles';
+import rules from './rules.react';
+
+type Props = {|
+  darkStyle: boolean,
+  children: string,
+|};
+function MessageMarkdown(props: Props) {
```

Comment:
Let's just call this `Markdown` instead of `MessageMarkdown`, since none of it is specific to messages. I think we'll want to use this same component for calendar entries down the line :)